Package.swift

```diff
@@ -16,7 +16,8 @@ let package = Package(
     ],
     products: [
         .executable(name: "xcstrings-tool", targets: ["xcstrings-tool"]),
-        .plugin(name: "XCStringsToolPlugin", targets: ["XCStringsToolPlugin"])
+        .plugin(name: "XCStringsToolPlugin", targets: ["XCStringsToolPlugin"]),
+        .library(name: "XCStringsToolStringCatalog", targets: ["StringCatalog"])
```

Comment:
I don't remember exactly what `name` does here, but I think it would be better to match the target name as that is consistent with the other two products

```suggestion
        .library(name: "StringCatalog", targets: ["StringCatalog"])
```